gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -587,60 +628,86 @@ def add_variant_annotations(
     """
     Add variant annotations to the MatrixTable.
 
-    These annotations included AC, AF, AN information split by homoplasmic/heteroplasmic, haplogroup, population, as well as histograms for quality metrics
+    These annotations include information such as AC, AF, AN split by homoplasmic/heteroplasmic.
 
     :param input_mt: MatrixTable
```

Comment:
```suggestion
    :param input_mt: Input MatrixTable
```

Reply:
I just realized all of your functions just have MatrixTable so this isn't needed, but if you do change it, I guess I recommend you change them all

Reply:
OK, will just keep as MatrixTable for now